testsuite/integration-arquillian/servers/app-server/jetty/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+~ Copyright 2018 Red Hat, Inc. and/or its affiliates
+~ and other contributors as indicated by the @author tags.
+~
+<?xml version="1.0"?>
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <parent>
+        <groupId>org.keycloak.testsuite</groupId>
+        <artifactId>integration-arquillian-servers-app-server</artifactId>
+        <version>4.8.3.Final-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>integration-arquillian-servers-app-server-jetty</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>common</module>
+        <module>81</module>
+        <module>94</module>
```

Comment:
I think we should include the modules in profile definition, it would build all jetty app servers each time the testsuite is built. 

Check https://github.com/keycloak/keycloak/blob/22e160136c37d73b05b7f4d77de5a2d1fcb7cef5/testsuite/integration-arquillian/servers/app-server/jboss/pom.xml#L710-L785

Reply:
+1

Reply:
I'm not sure if this isn't a bit over the top for Jetty. All modules build in less than a second:

```
[INFO] App Server - Jetty Parent .......................... SUCCESS [  0.024 s]
[INFO] App Server - Jetty Common .......................... SUCCESS [  0.512 s]
[INFO] App Server - Jetty 9.2 ............................. SUCCESS [  0.196 s]
[INFO] App Server - Jetty 9.3 ............................. SUCCESS [  0.195 s]
[INFO] App Server - Jetty 9.4 ............................. SUCCESS [  0.161 s]
```

Please let me know if you still would like to have this.

Reply:
But have you tested it without the dependencies downloaded to your local maven repo? I know the server is not that big as a wildfly but still.

Reply:
This is a very good point actually. Yes, I'll implement this shortly.

Reply:
Implemented. Thanks!